### PR TITLE
some general improvements

### PR DESCRIPTION
**Pull request by antgonza**
In this PR:
- add message about help to bootstrap messages
- add message about help to the error pages
- remove metadata search from html
- adds link to current/future features

### CODE REVIEW

File: qiita_pet/static/js/qiita.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ function bootstrapAlert(message, severity, timeout){
   // prepend a space to separate the message from the '!' icon
   alertDiv.append(' '+message);
 
+  // prepend the "Need help" message
+  if (severity == 'warning' || severity == 'danger'){
+    alertDiv.append(' </br></br>Need help? Send us an <a href="qiita.help@gmail.com">email</a>.');
```
Comment (squirrelo) [inline qiita_pet/static/js/qiita.js]: instead of `</br>` you can use `<p>STUFF</p>` for cleaner HTML.

File: qiita_pet/templates/error.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
 {% block content%}
     <h1 style="text-align:center">ERROR: CODE {{status_code}}!</h1>
     <p style="text-align:center">The error has been logged and will be looked at.</p>
+    <p style="text-align:center">Need help? Send us an <a href="qiita.help@gmail.com">email</a>.</p>
```
Comment (squirrelo) [inline qiita_pet/templates/error.html]: missing `mailto:` before email

File: qiita_pet/static/js/qiita.js [hunk 3]
```diff
@@ -21,6 +21,11 @@ function bootstrapAlert(message, severity, timeout){
   // prepend a space to separate the message from the '!' icon
   alertDiv.append(' '+message);
 
+  // prepend the "Need help" message
+  if (severity == 'warning' || severity == 'danger'){
+    alertDiv.append('<p style="text-align:center">Need help? Send us an <a href="mailto:qiita.help@gmail.com">email</a>.</p>');
```
Comment (squirrelo) [inline qiita_pet/static/js/qiita.js]: does not need `style="text-align:center"` unless we want that centered with the rest of the text left justified.

Comment (antgonza) [inline qiita_pet/static/js/qiita.js]: I'm OK having it centered ... do you want me to change it?

File: qiita_pet/templates/sitebase.html [hunk 4]
```diff
@@ -244,6 +244,11 @@
               </ul>
             </li>
           </ul>
+          <ul class="nav navbar-nav">
+            <li>
+              <a href="https://github.com/biocore/qiita/blob/master/README.rst#current-features">Current and Future Features</a>
```
Comment (squirrelo) [inline qiita_pet/templates/sitebase.html]: This is probably better under `More Info` instead of its own header link.

**Comment by squirrelo** [comment]
Couple more comments.

File: qiita_pet/templates/sitebase.html (hunk 4 above)
Comment (antgonza) [inline qiita_pet/templates/sitebase.html]: The idea is to have it as visible as possible for users. Thus, a separate direct link.

File: qiita_pet/static/js/qiita.js (hunk 3 above)
Comment (squirrelo) [inline qiita_pet/static/js/qiita.js]: No strong feelings either way.

**Comment by josenavas** [comment]
:+1:

**Pull request merged**